iron-multi-selectable.html

```diff
@@ -61,11 +61,9 @@
         if (this.selectedValues) {
           this._toggleSelected(value);
         } else {
-          this.selectedValues = [value];
+          this.set('selectedValues',[value]);
```

Comment:
Once I made the change to explicitly call the setter, the notifications were being called. I believe firefox was misbehaving.